release/README.md

```diff
@@ -19,13 +60,43 @@ the release binaries following these steps:
 
 1. `git clone https://github.com/btcsuite/btcd.git`
 2. `cd btcd`
-3. `./build/release/release.sh <TAG> # <TAG> is the name of the next
-   release/tag`
+3. `./release/release.sh <TAG> # <TAG> is the name of the next release/tag`
 
 This will then create a directory of the form `btcd-<TAG>` containing archives
 of the release binaries for each supported operating system and architecture,
 and a manifest file containing the hash of each archive.
 
+### Pushing a release (for maintainers)
+
+Now that the directory `btcd-<TAG>` is created, the manifest file needs to be
+signed by a maintainer and the release files need to be published to GitHub.
+
+Sign the `manifest-<TAG>.txt` file like so:
+```sh
+gpg --output manifest-<TAG>.txt.sig --sign manifest-<TAG>.txt
```

Comment:
I usually sign with a _detached_ sig. This produces a smaller output (as the document itself isn't included in the signature). Based on your example above, signing with a detached sig would look something like: 
```
gpg --sign --detach-sig manifest-<TAG>.txt
```

We can drop the `--output`, as it's automatically create the sig in a file name that adds `.sig` to the referenced file to be signed.

Reply:
Changed to detached sig, should be good now